src/core/__tests__/export-data.js

```diff
@@ -0,0 +1,621 @@
+var metadataForExport = {
```

Comment:
Where did all these data come from?

In general, I'm not a fan of the testing method of committing a 500+ line "good" dataset and then testing against it. If one of the tests stops working at some point, then rather than being able to look at the test and see a clear mismatch, we'll have to go digging through the (supposedly) good dataset and try and determine why it's not good any more.

It might be better than *not* testing, but it also might end up being harder to work with in the future.